### PR TITLE
Add padding to the text below 767px on page : /docker-extension-meshery

### LINKED ISSUE #3324: No padding to the text on page: /docker-extension-meshery
#### Description
No padding to text on page: /docker-extension-meshery
Screenshot 2022-10-15 103903

#### Expected Behavior
Screenshot 2022-10-15 104221

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by tinniaru3005**
**Description**
Fixed the issue by adding a padding of 15 px to left and right

This PR fixes #3324 
Closes #3324 
**Notes for Reviewers**
layer5

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by tinniaru3005** [comment]
Hey @Nikhil-Ladha, is it fine now?

File: src/sections/Docker-Meshery/docker-extension-meshery.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const DockerExtensionMeshery = () => {
           <h2 className="sectionTitle">
             Develop service mesh-ready apps using Docker Desktop and Meshery
           </h2>
-          <p>
+          <p className="sectionContent">
```
Comment (Nikhil-Ladha) [inline src/sections/Docker-Meshery/docker-extension-meshery.js:27]: This className should be applied to the `Row` component, as the header should also be padded. 
Though, a better fix for this is to set `margin: auto` in the CSS for the new class, that would fix the issue.

**Comment by l5io** [comment]
🚀 Preview for commit a50ebe6591da603d07b82d51d3ab9af26af2fe55 at: https://634e463a83fa50180b8ae2e5--layer5.netlify.app

**Comment by tinniaru3005** [comment]
Hey @Nikhil-Ladha, thank you so much for helping me out in solving the issue. Can you please let me know if the changes look fine now?

File: src/sections/Docker-Meshery/docker-extension-meshery.js (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/sections/Docker-Meshery/docker-extension-meshery.js:27]: Remove the className from here it is not required anymore.

**Comment by l5io** [comment]
🚀 Preview for commit cafc58edaf7881c26e0c9c41771db52578d1d622 at: https://634e9bdf16d79565cd8bfd46--layer5.netlify.app

File: src/sections/Docker-Meshery/docker-extension-meshery.js (hunk 1 above)
Comment (tinniaru3005) [reply src/sections/Docker-Meshery/docker-extension-meshery.js]: > Remove the className from here it is not required anymore.

Yes done :)

**Comment by tinniaru3005** [comment]
Hey @Nikhil-Ladha, I have made the changes. Can you please confirm if they are fine now?

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**